zokrates_parser/src/zokrates.pest

```diff
@@ -77,7 +77,9 @@ unary_expression = { op_unary ~ term }
 
 assignee = { identifier ~ ("[" ~ range_or_expression ~ "]")* }
 identifier = @{ ((!keyword ~ ASCII_ALPHA) | (keyword ~ (ASCII_ALPHANUMERIC | "_"))) ~ (ASCII_ALPHANUMERIC | "_")* }
-constant = @{ "0" | ASCII_NONZERO_DIGIT ~ ASCII_DIGIT* }
+constant = { decimal_number | boolean_constant }
+decimal_number = @{ "0" | ASCII_NONZERO_DIGIT ~ ASCII_DIGIT* }
+boolean_constant = { "true" | "false" }
```

Comment:
In the grammar, these should probably be called `boolean_literals`. Unimportant detail, though.

Reply:
Agreed